gc/base/standard/ConfigurationGenerational.cpp

```diff
@@ -219,4 +219,20 @@ MM_ConfigurationGenerational::defaultMemorySpaceAllocated(MM_GCExtensionsBase *e
 	extensions->setGuaranteedNurseryRange(nurseryStart, (void*)UDATA_MAX);
 }
 
+uintptr_t
+MM_ConfigurationGenerational::calculateDefaultRegionSize(MM_EnvironmentBase *env)
+{
+	uintptr_t regionSize = STANDARD_REGION_SIZE_BYTES;
+
+#if defined(OMR_GC_CONCURRENT_SCAVENGER)

```

Comment:
I am talking about fields in GCExtensionsBase.hpp
#if defined(OMR_GC_CONCURRENT_SCAVENGER)
	bool concurrentScavenger;	/* flag for check is Concurrent Scavenger active */
	bool debugConcurrentScavengerPageAlignment;
	uintptr_t concurrentScavengerPageSectionSize; /* selected section size for Concurrent Scavenger Page */
	void *concurrentScavengerPageStartAddress; /* start address for Concurrent Scavenger Page, UDATA_MAX if it is not initialized */
#endif /* defined(OMR_GC_CONCURRENT_SCAVENGER) */